src/nxt_main.c

```diff
@@ -5,11 +5,7 @@
  */
 
 #include <nxt_main.h>
-#include <nxt_runtime.h>
```

Comment:
Ugh, how did I miss that first time around! I blame GitHub!, now that I'm looking at the commits in a terminal it's so obvious!

It's been a while since I brought this old chestnut up, but doing patch review over email would be so much better!